src/tests/JIT/HardwareIntrinsics/Arm/Shared/Helpers.cs

```diff
@@ -6084,6 +6124,25 @@ private static ulong Pairwise(Func<ulong, ulong, ulong> pairOp, ulong[] op1, ulo
 
         public static float AddPairwise(float[] op1, float[] op2, int i) => Pairwise(Add, op1, op2, i);
 
+        public static float[] AddRotateComplex(float[] op1, float[] op2, byte rot)
+        {
+            for (int i = 0; i < op1.Length; i += 2)
+            {
+                if (rot == 0)
```

Comment:
likewise here - from https://developer.arm.com/architectures/instruction-sets/intrinsics/svcadd[_f64]_m

can we use `real` and `img` and then calculate the result to make it clear.

Reply:
seems this is missed.